src/Catty/PlayerEngine/Sensors/Phiro/PhiroBottomRightSensor.swift

```diff
@@ -43,7 +43,7 @@
         self.getBluetoothService()?.getSensorPhiro()?.getSensorValue(type(of: self).pinNumber) ?? type(of: self).defaultRawValue
     }
 
-    func convertToStandardized(rawValue: Double) -> Double {
+    func convertToStandardized(rawValue: Double, landscapeMode: Bool) -> Double {
```

Comment:
Please add a few tests to `PhiroSensorTest` to verify the `convertToStandardized` method.

Reply:
Done!